docs/tools/sdks/contract-sdks.mdx

```diff
@@ -45,3 +45,11 @@ soroban-sdk = { version = $VERSION, features = ["testutils"] }
 The `as-soroban-sdk` is an open source SDK that supports writing programs for the Soroban smart contract platform by using the AssemblyScript programming language.
 
 The AssemblyScript Soroban SDK is maintained by dedicated community developer, Soneso. Report issues and share feedback [here](https://github.com/Soneso/as-soroban-sdk/issues/new).
+
+### Stellar Axelar Std Derive Rust Crate
+
+Axelar has created a Rust crate with useful macros for Stellar smart contract development. Please see Rust Crate [`stellar_axelar_std_derive`](https://axelarnetwork.github.io/axelar-amplifier-stellar/stellar_axelar_std_derive/index.html) for Attribute Macros and Derive Macros, and additional information.
+
+Please note that in the future, some macros will be moved into Soroban Rust SDK.
+
+Please also note, some macros such as `pausible` and `upgradable` also exist in the OpenZeppelin for Stellar Library: https://github.com/OpenZeppelin/stellar-contracts.
```

Comment:
Could we create a section for OpenZeppelin here too instead of the brief mention?

Reply:
I like the idea of including OZ's crates here, too. But, have they published them yet? [Their README](https://github.com/OpenZeppelin/stellar-contracts/tree/main?tab=readme-ov-file#usage) suggests not yet. That might not even be a deal-breaker, but wanted to ask your thoughts?

Reply:
We can omit mention of the OZ lib until published.

Reply:
I will mention that it is coming soon and replace the language when the crates exist.